app/src/main/java/com/aileyzhang/musicapp/views/SongListItemView.java

```diff
@@ -0,0 +1,26 @@
+package com.aileyzhang.musicapp.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.aileyzhang.musicapp.R;
+
+/**
+ * Created by Ailey on 2017-12-18.
+ */
+
+public class SongListItemView extends View {
+    View view;
+
+    public SongListItemView(Context context, ViewGroup parent) {
```

Comment:
Instead of this custom constructor, override the one of the two default constructors with signature `(Context context)` and `Context context, AttributeSet attrs)`. Here's more reference on this: http://blog.danlew.net/2016/07/19/a-deep-dive-into-android-view-constructors/

Also you can just use the view class's `inflate(...)` method without needing to get the layout inflater using the context.